README.md

```diff
@@ -1,3 +1,107 @@
-# LearnGit
-Just Learning Git
+# CommandLineParser
+# Introduction
+[1].CommandLineParser library adds the user defined command line options which accepts the short command , long command , short description ,long description and a  default value.
 
+[2].This library parses the command line options and gives the corresponding value of a particular command.
+
+[3].The CommandLine parser Library supports custom help functionlity .
+
+[4].The command line Options can be of the following forms :- 
+
+    --port=3132
+    --port 3132
+    -p
+    -cp
+    -p 4215
+    
+[5].Example 
+
+let say the, resgistred command is port . 
+ 
+    obj.Addoptions("p", "port", "enter the port numbers", "port number must be of 4 digits",1234);
+ 
+In  CommandPrompt
+
+    SampleApp.exe --port=324 425
+ 
```

Comment:
obj.Addoptions says port number must be 4 digits. But in example it seems to be different